src/services/TeamHub/TeamHub.BLL/Contracts/ITaskHandlerService.cs

```diff
@@ -0,0 +1,18 @@
+using TeamHub.DAL.Models;
+
+namespace TeamHub.BLL.Contracts
+{
+    public interface ITaskHandlerService
+    {
+        Task<TaskHandler> AddTaskHandlerAsync(
+            TeamMember teamMember,
+            int taskId,
+            CancellationToken cancellationToken
+        );
+        Task<TaskHandler> RemoveTaskHandlerAsync(
+            TeamMember teamMember,
+            int taskId,
+            CancellationToken cancellationToken
+        );
```

Comment:
You don't need to create your own services for this. Describe the implementation of creation and deletion in the handler for your commands